### ISSUE TITLE
Unable to Create Kubernetes Cluster

### ISSUE THREAD

**Issue by rawkode** [pulumi/pulumi-civo#65]
The `pulumi up` does create a cluster, as I can see it from the UI. However, Pulumi fails to finish the `up` and errors with the message:

```
Diagnostics:
  pulumi:pulumi:Stack (rudder-production):
    error: update failed

  civo:index:KubernetesCluster (cluster):
    error: 1 error occurred:
    	* updating urn:pulumi:production::rudder::civo:index/kubernetesCluster:KubernetesCluster::cluster: 1 error occurred:
    	* error waiting for cluster (868b6c70-1431-4cc3-bb38-9b2431f1f218) to be created: UnknownError
```

This has happened 5/5 attempts.

Code:

```typescript
import * as pulumi from "@pulumi/pulumi";
import * as civo from "@pulumi/civo";

const kubernetesCluster = new civo.KubernetesCluster("cluster", {
  kubernetesVersion: "1.21.0+k3s1",
  region: "lon1",
  numTargetNodes: 3,
  targetNodesSize: "g3.k3s.small",
});
```

**Comment by rawkode** [comment]
Running `pulumi up` again, after the error returns:

```
error: 1 error occurred:
	* updating urn:pulumi:production::rudder::civo:index/kubernetesCluster:KubernetesCluster::cluster: 1 error occurred:
	* error waiting for cluster (868b6c70-1431-4cc3-bb38-9b2431f1f218) to be created: unexpected state 'BUILDING', wanted target 'ACTIVE'. last error: %!s(<nil>)
```

**Comment by alejandrojnm** [comment]
i will check this

**Comment by stack72** [comment]
@rawkode try and remove the kubernetes version - I've had similar experiences for this

**Comment by alejandrojnm** [comment]
i found a bug in the provider, I'm working on that

**Comment by alejandrojnm** [comment]
This is already fixed in the las version (v0.10.3)

**Comment by stack72** [comment]
Thanks so much @alejandrojnm - I'll pull this in first thing in the morning
